src/atomate2/common/jobs/phonons.py

```diff
@@ -153,11 +153,13 @@ def generate_phonon_displacements(
         "of the symmetry of the structure and thus will be removed now.",
         stacklevel=1,
     )
-    cell = get_phonopy_structure(
+
+    # Note that `remove_site_property` returns None
```

Comment:
this is no longer true. i changed a lot of structure methods to return self 3 weeks ago in https://github.com/materialsproject/pymatgen/pull/3623